conf/cassandra.yaml

```diff
@@ -566,6 +566,35 @@ commitlog_segment_size: 32MiB
 #
 # flush_compression: fast
 
+
+# Defines the compression params used to read/write the sstables.  If undefined the default is LZ4 with 16kb chunk
+# length and min_compress_ratio of 0.0 (see CompressionParams.DEFAULT_MIN_COMPRESS_RATIO).
+# To disable compression set `type: none` or set the `enable` parameter as `false`.
+# This is the default options for sstable compression, the (schema) table-level settings will override this option.
+#sstable_compressor:
+#
+  #enable=false # defaults to true
+#
+  #chunk_length: 16KiB : defaults to 16KiB
+#
+  # min_compress_ratio: 0.0
+  # max_compressed_length: 16KiB
```

Comment:
For readability, make the example standalone, add comments afterwards.
```suggestion
#
# sstable_compressor:
  # enable=true 
  # chunk_length: 16KiB
  # min_compress_ratio: 0.0
  # max_compressed_length: 16KiB
  # - class_name:
  # parameters:
     # - param1 : value
#     
# 
# enable defaults to true 
# chunk_length defaults to 16KiB
# Only one of the min_compress_ratio and max_compressed_length options can be specified.  
# The mathematically related in that
# chunk_length / max_compressed_length = min_compress_ratio
# if neither option is specified a min_compress_ratio of 0.0 is the default.
# A min_compress_ratio <= 0 implies max_compressed_length of Integer.MAX_VALUE KiB
# The class_name is the compressor class name. It may be one of the following aliases:
# lz4,  none,  noop,  snappy,  deflate,  or zstd; or it the fully qualified  name of a class
# that implements ICompressor
```